packages/experience/src/components/LoadingMask/index.module.scss

```diff
@@ -0,0 +1,8 @@
+@use '@/scss/underscore' as _;
+
+.overlay {
+  position: fixed;
+  inset: 0;
+  @include _.flex-column;
+  z-index: 300;
```

Comment:
Maybe we should also manage the z-index values in an enum. Like what we did in console

Reply:
Good idea, will add in another PR.

Reply:
LOG-9644